frontend/components/events/PendingEvents.jsx

```diff
@@ -27,38 +29,57 @@ const PendingEvents = ({DYNAMICUSERINFO}) => {
     // perform axios request
   }
 
-    return (
-      <View style={styles.container}>
-        {pendingEvents.map((each) => {
-          const unformattedDate = each.eventDate;
-          const niceDate = format(unformattedDate, 'MMM do, yy')
-          const niceTime = format(unformattedDate, 'h:mmaa');
-          return (
-            <View style={styles.singleEvent} key={each.eventId}>
-              <Text style={styles.name}>{each.eventTitle}</Text>
-              <Text style={styles.host}>{each.hostUsername}</Text>
-              <Text style={styles.location}>{each.eventLocation}</Text>
-              <Text style={styles.datetime}>{niceDate}</Text>
-              <Text style={styles.datetime}>{niceTime}</Text>
-              <Text style={styles.description}>{each.eventDescription}</Text>
-              <Button title="Accept" onPress={() => {handleAccept(each.eventId)}}/>
-              <Button color="#FF0000" title="Reject" onPress={() => {handleReject(each.eventId)}}/>
-            </View>
-          )
-        })}
-      </View>
-    )
-  }
-  export default PendingEvents;
+  return (
+    <View style={styles.container}>
+      {pendingEvents.map((each) => {
+        const [loading, setLoading] = useState(true)
+        const [address, setAddress] = useState(null);
+        const unformattedDate = each.eventDate;
+        const niceDate = format(unformattedDate, 'MMM do, yy')
+        const niceTime = format(unformattedDate, 'h:mmaa');
+        const geocodingInput = {
+          latitude: each.eventLocation[0],
+          longitude: each.eventLocation[1]
+        };
+        useEffect(() => {
+          Location.reverseGeocodeAsync(geocodingInput)
+          .then((geocodedAddress) => {
+            setAddress(
+              `${geocodedAddress[0].streetNumber ? `${geocodedAddress[0].streetNumber} ` : ''}${geocodedAddress[0].street}, ${geocodedAddress[0].city} ${geocodedAddress[0].region} ${geocodedAddress[0].postalCode}`
+            );
+          })
+          .catch((err) => {
+            console.log(err);
```

Comment:
Remove console.log. AJAX errors should only be caught on the front end if they're going to cause the request to retry or some other flow of logic